cmd/cloud/user.go

```diff
@@ -0,0 +1,59 @@
+package cloud
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+)
+
+var role string
+
+func newUserCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "user",
+		Aliases: []string{"us"},
+		Short:   "Invite Users to your Astro Organization",
+		Long:    "Invite Users to your Astro Organization.",
+	}
+	cmd.SetOut(out)
+	cmd.AddCommand(
+		newUserInviteCmd(out),
+	)
+	return cmd
+}
+
+func newUserInviteCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "invite",
+		Aliases: []string{"inv"},
+		Short:   "Invite a user to your Astro Organization",
+		Long:    "astro user invite [email] --role [ORGANIZATION_MEMBER, ORGANIZATION_BILLING_ADMIN, ORGANIZATION_OWNER].",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return userInvite(cmd, args, out)
+		},
+	}
+	cmd.Flags().StringVarP(&role, "role", "r", "ORGANIZATION_MEMBER", "The invited user's role. It can be one of ORGANIZATION_MEMBER, ORGANIZATION_BILLING_ADMIN or ORGANIZATION_OWNER ")
+	return cmd
+}
+
+func userInvite(cmd *cobra.Command, args []string, out io.Writer) error {
+	var email, outMsg string
+	cmd.SilenceUsage = false
+	if len(args) < 1 {
+		_, err := out.Write([]byte("valid email is required to create an invite"))
+		if err != nil {
+			return err
+		}
+	}
+
+	email = args[0]
+	// TODO: call client.CreateUserInvite() with the correct input
+	// TODO: should client.CreateUserInvite() handle output or the command?
```

Comment:
The usual pattern is that client.CreateUserInvite() would handle the output

Reply:
@sunkickr Since the command is asking for input, I have left the output printing here as well. Let me know if this is ok and if we really want to handle output from `client.CreateUserInvite()` instead

Reply:
changed it.